scripts/start-deployQuilt

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+set -eu
+
+# shellcheck source=start-utils
+. "${SCRIPTS:-/}start-utils"
+
+requireVar VANILLA_VERSION
+export TYPE=QUILT
+: "${QUILT_LAUNCHER_VERSION:=${QUILT_INSTALLER_VERSION:-LATEST}}"
+: "${QUILT_LAUNCHER:=}"
+: "${QUILT_LAUNCHER_URL:=}"
+: "${QUILT_INSTALLER_VERSION:=LATEST}"
+: "${QUILT_SERVER_VERSION:=${VANILLA_VERSION}}"
+
+isDebugging && set -x
+
+# Custom quilt jar
+if [[ $QUILT_LAUNCHER ]]; then
+  export SERVER=${QUILT_LAUNCHER}
+# Custom quilt jar url
+elif [[ $QUILT_LAUNCHER_URL ]]; then
+  export SERVER=quilt-server-$(echo -n "$QUILT_LAUNCHER_URL" | mc-image-helper hash)
+# Official quilt installer
+else
+  if [[ ${QUILT_INSTALLER_VERSION^^} = LATEST ]]; then
+    log "Checking Quilt Loader version information."
+    QUILT_INSTALLER_VERSION=$(maven-metadata-release https://maven.quiltmc.org/repository/release/org/quiltmc/quilt-installer/maven-metadata.xml)
+  fi
+  export INSTALLER=quilt-installer-${QUILT_INSTALLER_VERSION}.jar
+  export SERVER=quilt-server-launch.jar 
```

Comment:
It would be good to qualify the file name by the server version to enable in place upgrades. If the installer doesn't let you override that, then you could rename the jar to the versioned name.